### PR TITLE
Adding support for requesting jobs be run

### PR DESCRIPTION
**Pull request by mcginleyr1**
Still todo allow api endpoint override.

### CODE REVIEW

**Comment by mcginleyr1** [comment]
Introducing as a new .py file with it's own args to ensure backwards compatibility.  We might rename it zodiac-jobs.py not sure yet still thinking.

File: api.py [hunk 1]
```diff
@@ -53,10 +54,28 @@ def upload_file(self, filepath, description):
...
             {'description': description, 'filename': filename, 's3_path': s3_path}
         )
+        return s3_path
+
+
+    def submit_job(self, args):
+        txlogs = []
+        attrfiles = []
+        for trans_log in args.transactions:
+            txlogs.append(self.upload_file(trans_log, trans_log))
```
Comment (sbeverly) [inline api.py]: Do we always want the description to just be a file path?  Would that ever be confusing if they don't name things clearly?

Comment (mcginleyr1) [inline api.py]: Descriptions gonna die. this is gonna stop being  dumping ground for shit at some point.

**Comment by djworth** [comment]
👍

**Pull request merged**